addons/io_scene_gltf2/io/com/gltf2_io_draco_compression_extension.py

```diff
@@ -28,6 +41,7 @@ def dll_path() -> Path:
     python_version = 'python{v[0]}.{v[1]}'.format(v=sys.version_info)
 
     path = os.environ.get('BLENDER_EXTERN_DRACO_LIBRARY_PATH')
+    path = find_draco_dll_in_module()
```

Comment:
This prevents BLENDER_EXTERN_DRACO_LIBRARY_PATH from working. Perhaps you meant to run it only if path was None.